src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,23 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
+
 public class Actor implements Cloneable {
```

Comment:
Implementing Cloneable and overriding the clone method is not a common practice for JPA entities. Cloning entities can lead to issues with entity identity and unit of work patterns. If you need to create a copy of an entity, consider using copy constructors or a dedicated method that clearly indicates the purpose of the copy.